helm/fmtok8s-email-rest/values.yaml

```diff
@@ -3,54 +3,22 @@
 # Declare variables to be passed into your templates.
 replicaCount: 1
 image:
-  repository: salaboy/fmtok8s-email-service
+  repository: matheuscruzdev/fmtok8s-email-service
```

Comment:
Let's change this to `ghcr.io/salaboy/fmtok8s-email-service` right? We need the one hosted in github packages

Reply:
@mcruzdev I think we need: `ghcr.io/` because this is going to go to docker hub if we don't add the registry